pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -75,6 +75,27 @@ def test_paths(self):
             "hello from B",
         ]
 
+    def test_paths_from_packages(self):
+        self.writefile("utils/__init__.py", "")
+        self.writefile("utils/a.py", "x = 'hello from A'")
+        self.pyscript_run(
+            """
+            <py-config>
+                paths = ["./utils/__init__.py", "./utils/a.py"]
```

Comment:
if we get paths without __init__.py, should we make one?

Reply:
You mean, if we only have `./utils/a.py` for example?

I think yes, it is better to always remain consistent aka we retain the directory structure that you provide us.
Even if there is no `__init__.py` present.

That way, there is no mismatch in what a user expects. Thus, even in the case of just `./utils/a.py`, it will create the `./utils` directory.

Reply:
[question] In terms of local modules, I assume we need a `__init__.py` even if the file is empty?

Reply:
no, it's conceptually wrong to create an `__init__.py` if it's not specified, for two reasons:
1. nowadays python can import packages even without an `__init__.py` and it has a slightly different behavior so adding one can change behavior
2. sometimes `__init__.py` are not empty: it might be possible that the user just forgot to list `__init__.py` in the `paths=...`, and in that case it's much nicer to let them see a clear failure than to create an empty `__init__.py` which would cause many headaches.